docs/api.md

```diff
@@ -1040,6 +1044,14 @@ to use the API after this request has succeeded.
   <!--end-request-body-post-accountdevicedestroy-id-->
 
 
+#### GET /recovery_email/enabled
+
+:lock: HAWK-authenticated with session token
+<!--begin-route-get-recovery_emailenabled-->
+Returns whether or not secondary emails is enabled for a user.
+<!--end-route-get-recovery_emailenabled-->
```

Comment:
I like the approach @vbudhram.  My one nit with the naming here is that the `/recovery_email` namespace is hugely overloaded, and it's not obvious that the "enabled" state refers only to a subset of its available functionality, i.e. the ability to add secondary email addresses.

Do we intend for this endpoint to eventually go away once we've completed rollout of the feature?  If so, perhaps we can name it something *really* specific like `/recovery_email/check_can_add_secondary_address` so that (a) future us won't be confused about what it checks, and (b) we'll be encouraged to ultimately remove it.

If we need to keep it longer-term, maybe we should take a more holisitic look at how to agree on feature state between auth-server and content-server.